stdlib/typing_extensions.pyi

```diff
@@ -670,6 +686,9 @@ else:
         globals: Mapping[str, Any] | None = None,  # value types depend on the key
         locals: Mapping[str, Any] | None = None,  # value types depend on the key
         type_params: Iterable[TypeVar | ParamSpec | TypeVarTuple] | None = None,
-        format: Format = Format.VALUE,  # noqa: Y011
+        format: Format | None = None,
```

Comment:
Is this change correct?

Reply:
Yes. VALUE is still the effective default but I made it not the runtime default to avoid importing annotationlib in typing.py.